src/shared/types/ReactPropTypeLocationNames.js

```diff
@@ -16,14 +16,14 @@ import type { ReactPropTypeLocations } from 'ReactPropTypeLocations';
 
 type NamesType = {[key: ReactPropTypeLocations]: string};
 
-var ReactPropTypeLocationNames = {};
+var ReactPropTypeLocationNames: NamesType = {};
```

Comment:
Previously, the exported type seemed to be "object literal" and I couldn't index into it with a ReactPropTypeLocations.